File: cashier-photocopy.py
hrg = 225 #cost per sheet
print("     ***Photocopy Cost App***     ")
print("INPUT DATA")
nm = input("Consument Name : ")
lbr = input("Many of Sheet : ")
lbr = int(lbr)
by = lbr*hrg
if lbr<100:
    ptg=(5*by)/100
elif lbr>=100 and lbr<=300:
    ptg=(15*by)/100
else:
    ptg=(30*by)/100

print("THE CALCULATION RESULT")
print("Consument Name : "+ nm)
print("Many of Sheet :"+ str(lbr))
print("Photocopy Cost :"+ str(by))
print("Discount :"+ str(ptg))
print("====================")
print("Total : "+ str(by-ptg))

      
    
           

File: cashier-photocopy.cpp
#include<stdio.h>
#include<conio.h>
#include<iostream>
using namespace std;

main(){
	string nm;
	int hrg=225;
	int lbr;
	float ptg, by;
	cout<<"     ***Aplikasi Ongkos Fotocopy***     \n\n\n"<< endl;
	cout<< "INPUT DATA"<<endl;
	cout<< "Nama Konsumen : ";
	getline(cin,nm);
	cout << "Banyaknya Lembar : ";
	cin >> lbr;
	by=lbr*hrg;
	if(lbr<100){
		ptg=(5*by)/100;
	}
	else if(lbr >= 100 && lbr <= 300) {
		ptg=(15*by)/100;
	}
	else {
		ptg=(20*by)/100;
	}
	cout << "\n\nHASIL PERHITUNGAN" << endl;
	cout << "Nama Konsumen : "<< nm << endl;
	cout << "Banyaknya Lembar : "<< lbr << endl;
	cout << "Biaya Fotocopy : Rp."<< by << endl;
	cout << "Potongan : Rp."<< ptg << endl;
	cout << "============================"<< endl;
	cout << "Total Pembayaran : Rp."<< by-ptg << endl;
	getch();
}
 
